sotoki/sotoki.py

```diff
@@ -1367,6 +1435,18 @@ def run():
     print(
         "starting sotoki scraper...{}".format(f"using {TMPFS_DIR}" if TMPFS_DIR else "")
     )
+    if arguments["--optimization-cache"] is not None:
+        s3 = KiwixStorage(arguments["--optimization-cache"])
+        if not s3.check_credentials(
+            list_buckets=True, failsafe=True, write=True, read=True
+        ):
+            raise AuthenticationError(
+                "Bad authentication credentials supplied for optimization cache. Please check and try again."
+            )
+        print("Credentials checked. Using optimization cache")
```

Comment:
It's also important that the log gives us some information about which cache was used. Youtube does https://github.com/openzim/youtube/blob/master/youtube2zim/scraper.py#L311

``` py
logger.info(f"  using cache: {self.s3_storage.url.netloc}  bucket: {self.s3_storage.bucket_name}")
```

Reply:
Changed this as requested